Assignment/solution_student/about_me.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Joe Johnson About Me Page</title>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<meta charset="UTF-8">
+	</head>
+	<body>
+		<h3>About Me | <a href="portofolio.html">Portofolio<a/>
+		</h3>
+		<h1>JOE JOHNSON
+		</h1>
+		<hr>
+		<img src="images/avatar.png" alt="Joe Johnson" align="right"HSPACE="10" VSPACE="10"> 
+		<BR CLEAR="left"/>
```

Comment:
Avoid using the `<br>` tag for spacing, for the same reason as the other presentational markup, and instead achieve this with `padding` or `margin` in CSS